manifests/config.pp

```diff
@@ -86,4 +80,34 @@
       match => '^SELINUXTYPE=\w+',
     }
   }
+
+  # Module build config:
+  validate_absolute_path($module_build_root)
+
+  file {$module_build_root:
+    ensure => 'directory',
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0644',
+  }
+
+  # put helper in place:
+  file {"${module_build_root}/modules/selinux_build_module.sh":
```

Comment:
even if the puppet agent process runs as `puppetagent_t` type it does not produce an AVC when it executes the selinux_build_module.sh script which has type  `usr_t`. so this is fine. I just wonder why it does not.

by default the puppet agent process even does run in `unconfinged_service_t` (or `initrc_`t) because the paths in the refpolicy were not adapted to the /opt/puppetlabs change.